.github/workflows/continuous-integration.yml

```diff
@@ -25,7 +25,11 @@ jobs:
         # This provides a dedicated "check" to asynchronously test all integration tests within the tests/ directory
         # The project name from the tests/ directory is then re-used within the "qa" job to run the generated "container"
         # within that directory.
-        run: cd tests; echo "matrix=[\"$(ls -d * | tr '\n' ' ' | sed 's/ $//' | sed 's/ /\",\"/g')\"]" >> $GITHUB_OUTPUT
+        run: |
```

Comment:
Is this needed ASWELL as pinning the version of `jq-action`? I would hope this change would be all that's needed?

Reply:
No - But, if there's any chance that `matrix` _is_ multiline, at least the case is covered. It should probably always have been like this WRT #150 

We may be able to stick with `jq-action@v2` if the `multiline` option works out for us.

Reply:
I think this change needs reverting as `"matrix=[\"$(ls -d * | tr '\n' ' ' | sed 's/ $//' | sed 's/ /\",\"/g')\"]"` should be on a single line.

Reply:
I'll send in another patch @internalsystemerror for 1.23

Reply:
Ty <3